libs/deps/merch-offers.js

```diff
@@ -0,0 +1,95 @@
+// Tue, 21 Nov 2023 12:17:48 GMT
```

Comment:
Should we take the whole file content for granted or it's up for discussion?

Reply:
These are built using esbuild. So take for granted.
Feel free to review our merch tacocat PRs at https://git.corp.adobe.com/wcms/tacocat.js/pulls :)

Reply:
Thank you!